### PR TITLE
bug-fix: 9575461 - IsConcatSpreadable is called twice

### PR DESCRIPTION
**Pull request by obastemur**

### CODE REVIEW

File: lib/Runtime/Library/JavascriptArray.cpp [hunk 1]
```diff
@@ -3019,8 +3023,8 @@ namespace Js
         for (uint idxArg = start; idxArg < args.Info.Count; idxArg++)
         {
             Var aItem = args[idxArg];
-            BOOL spreadable = false;
-            if (scriptContext->GetConfig()->IsES6IsConcatSpreadableEnabled())
+            BOOL spreadable = spreadableCheckedAndTrue;
```
Comment (akroshg) [inline lib/Runtime/Library/JavascriptArray.cpp]: nit: change BOOL to bool as the spreadableCheckedAndTrue is bool?

**Comment by akroshg** [comment]
looks good to me.

File: lib/Runtime/Library/JavascriptArray.cpp (hunk 1 above)
Comment (obastemur) [inline lib/Runtime/Library/JavascriptArray.cpp]: > change BOOL to bool as the spreadableCheckedAndTrue is bool?

Right! Thanks!

**Comment by obastemur** [comment]
@akroshg Thanks for the review

**Pull request merged**